app/models/offer.rb

```diff
@@ -7,7 +7,8 @@ class Offer < ActiveRecord::Base
   include OfferSearch
   include OfferFiltering
 
-  NOT_ACTIVE_STATES = ["received", "closed", "cancelled", "inactive"]
+  NOT_ACTIVE_STATES = ["received", "closed", "cancelled", "inactive"].freeze
```

Comment:
Use `%w` or `%W` for an array of words. eg `%w[a b c d] => ["a", "b", "c","d"]`

Reply:
I'll confess I'm not a big fan of the `%w` ruby pattern. Because :
- I find the difference in readability to be minor
- it obfuscates the data type 
- it isn't explicit in what it does, unlike everything else in ruby with clear expressions such as `unless`, `do`
- it isn't standard across any other language so that creates an extra layer of confusion for ruby learners such as myself.

Maybe I'm just missing the point of it altogether 🤔
Ruby guru @steveyken , would you know actually is the point 😆

I'm fine with either way, just sharing my thoughts and trying to learn : )

Reply:
not sure on the etymology of %w.... sorry!